gnovm/pkg/gnolang/alloc.go

```diff
@@ -2,16 +2,21 @@ package gnolang
 
 import (
 	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/overflow"
+	"github.com/gnolang/gno/tm2/pkg/store"
 )
 
 // Keeps track of in-memory allocations.
 // In the future, allocations within realm boundaries will be
 // (optionally?) condensed (objects to be GC'd will be discarded),
 // but for now, allocations strictly increment across the whole tx.
 type Allocator struct {
-	maxBytes int64
-	bytes    int64
-	collect  func() (left int64, ok bool) // gc callback
+	maxBytes     int64
+	bytes        int64
+	maxBytesUsed int64
```

Comment:
or peakBytes for the name?

Reply:
yes right it is better name

Reply:
done here https://github.com/gnolang/gno/pull/4546/commits/3f5d12a3bb417edc14e541659a757592cb2eac26